torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3144,6 +3145,30 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            supports_out=False,
            sample_inputs_func=sample_inputs_max_min_reduction_no_dim,),
+    OpInfo('maximum',
+           op=torch.maximum,
+           dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCPU=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
```

Comment:
In an OpInfo, `dtypesIfCPU` and `dtypesIfCUDA` only need to be assigned, if they're different from the `dtypes` list. I see that these redundancies arose from looking at the `OpInfos` of `min` & `max`, that are close to these ones. I had introduced those redundancies after looking at the OpInfos of `masked_scattered`, etc, but would be removing them.

Reply:
That would be a nice cleanup, you're right, but let's get this merged.